packages/shared/src/components/modals/recruiter/RecruiterTrustModal.tsx

```diff
@@ -119,8 +118,11 @@ export const RecruiterTrustModal = ({
             Agree
           </Button>
 
-          <Link passHref href={`${webappUrl}`}>
-            <a className="text-center text-text-secondary underline typo-callout">
+          <Link passHref href="https://r.daily.dev/state-of-trust">
```

Comment:
Please put these in links constant, also you missed rel={defaultAnchorRel} below.
I think you can actually use TextLink or something like as a component, but that's optional.